src/Common/Commands.Common.Keyvault/Common.Keyvault.Netcore.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\tools\Common.Netcore.Dependencies.targets" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>Microsoft.WindowsAzure.Commands.Common.Keyvault</AssemblyName>
```

Comment:
@praries880 this should be `Microsoft.Azure.Commands.Common.KeyVault`

Reply:
I aped this off [storage](https://github.com/Azure/azure-powershell/blob/preview/src/Common/Commands.Common.Storage/Common.Storage.Netcore.csproj#L7) will fix it in my project... what about storage though?